experiments/pagination/README.md

```diff
@@ -0,0 +1,98 @@
+
+## Pager 
+On the [page](http://localhost:4502/content/searchtester/us/en/experiments/pager.html) you will see 4 fields which
+you may recognize as
+[Querybuilder](https://helpx.adobe.com/experience-manager/6-5/sites/developing/using/reference-materials/javadoc/com/day/cq/search/QueryBuilder.html)
+fields:
+
+* limit: Page Size
+* guessTotal: Whether to finish the full query to determine the total size (see the [Guess Total experiment](https://github.com/adobe/aem-search-experiments/tree/master/experiments/large-result-sets) for more information)
+* offset: Where, in the result list, to start extracting hits to return 
+* path: The base path where the search will begin
+
+After clicking _Load Page_, these fields will be updated, if required, in order to be ready for the next load.
+
+You will see 2 buttons:
+
+* Load Page: Use the fields' values to start a search of cq:Page items
+* Reset Fields: Reset all the values to their default values
+
+You will see 2 text area fields:
+
+* Results: The list of all the hits already returned with the most recent hits appended to the bottom. Scrolling to the bottom of the list will trigger the next load (i.e. infinite scroll).
+* Details: The details of the most recently executed search call including time duration, URL and the response.
+
+## Test #1: 
+1. Open Pager: http://localhost:4502/content/searchtester/us/en/experiments/pager.html
+1. Examine the fields (note the _Path_ is pointing to the pager location, to limit the number of hits)
+1. Click _Load Page_ button 
+1. Examine how the input fields changed 
+1. Examine the text area fields
+1. Note the _Details_, especially the Duration, how the URL matches the field values and the response metadata
+1. Click the _Load Page_ again and see the _Result_ list grow, and see the _Details_ information refresh
+
+## Test #2:
+1. Open Pager: http://localhost:4502/content/searchtester/us/en/experiments/pager.html or click the _Clear Fields_
+button if the page is already open
+1. Change the Path value to **"/content"** to work with a larger result set
+1. Click _Load Page_
+1. In _Details_, notice the Duration value
+1. Change the _guessTotal_ value to **"false"**
+1. Click _Load Page_
+1. In _Details_, notice the Duration value is substantially larger (relatively) because the backend finished the search in order to find the exact total
+
+## Test #3:
+1. Open the Pages page: http://localhost:4502/content/searchtester/us/en/experiments/pager.html or click the
+_Clear Fields_ button if the page is already open
+1. It is up to you.  Some ideas:
+    * Change the _Limit_ to see different page sizes and notice what happens to the _Offset_ field after a load
+    * Change the _guessTotal_ values to "true", "false", -1, 0, 1 and 10000 and check the _Details_ of that search
+    * Open your browser's network tab and see the network traffic when the call is executed
+    * Use **"/content"** as the path for a larger search set
+
+## Conclusion
```

Comment:
From @clotton 
The conclusion should probably link to official docs on the topic of pagination and invite the reader to explore the topic further.

Reply:
Not a lot to link too.  Can we link to non-Adobe docs?

Reply:
@top This'll work:  https://docs.adobe.com/content/help/en/experience-manager-64/developing/platform/query-builder/querybuilder-api.html#implementing-pagination

Reply:
That's the one I used at the bottom (minus the 'implementing-pagination' part - good addition)